typescript/src/entities/product.test.ts

```diff
@@ -23,13 +23,30 @@ test('can get and set name', () => {
   expect(product.name).toBe('example');
 });
 
+test('can get and set domain', () => {
+  const merchi = new Merchi();
+  const product = new merchi.Product();
+  const domain = new merchi.Domain();
+  product.domain = domain;
+  expect(product.domain).toBe(domain);
+});
+
 test('can fetch product from server', () => {
   const merchi = new Merchi();
   const testName = 'S7qHUfV_dr5l';
   mockFetch(true, {'product': {'name': testName}}, 200);
   return merchi.Product.get(1).then(product => expect(product.name).toBe(testName));
 });
 
+test('can fetch with explicit session token', () => {
+  const testToken = "YrDwzmh8&QGtAfg9quh(4QfSlE^RPXWl";
+  const merchi = new Merchi(testToken);
+  const testName = 'S7qHUfV_dr5l';
+  mockFetch(true, {'product': {'name': testName}}, 200);
+  (window as any).merchiBackendUri = 'http://override.example.com/';
```

Comment:
I think we have as setup thing and this is not needed any more?

Reply:
correct